packages/rehype-shikiji/test/fixtures/a.md

```diff
@@ -2,9 +2,10 @@
 
 …world!
 
-```js {3-4} fileName=test
+```js {3-4} fileName="/test/md" /a/
```

Comment:
Is this change indented?

Reply:
Oh the `/a/` is for the previous implementation, I will remove that later